src/installer/installer.go

```diff
@@ -176,6 +178,42 @@ func (i *installer) startBootstrap() error {
 		return err
 	}
 
+	i.log.Infof("Patching masters ignition with bootstrap public SSH key")
+	var mc types.MachineConfig
+	yamlFile := "/opt/openshift/openshift/99_openshift-machineconfig_99-additional-master-ssh.yaml"
+	fieldName := "Spec.Config.Passwd.Users[0].SshAuthorizedKeys"
+	newSshKey, err := i.ops.ReadFile("/etc/ssh/ssh_host_rsa_key.pub")
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+	_, err = i.ops.ExecCommand(utils.NewLogWriter(i.log), "cp", "/99_openshift-machineconfig_99-additional-master-ssh.yaml", "/opt/openshift/openshift/")
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+	err = i.patchMastersIgnition(yamlFile, fieldName, string(newSshKey), "put", &mc)
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+	i.log.Infof("Copying boostrap private SSH key to default .ssh directory on FS")
+	err = i.ops.Mkdir("/root/.ssh")
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
+	_, err = i.ops.ExecPrivilegeCommand(utils.NewLogWriter(i.log), "cp", "/etc/ssh/ssh_host_rsa_key", "/root/.ssh/id_rsa")
+	if err != nil {
+		i.log.Error(err)
+		return err
+	}
```

Comment:
I think that if you generate your own key pair this won't be required.
* by default it will get created under ~/.ssh with the proper name.
`chmod` won't be required as well

Reply:
done